projects/packages/account-protection/src/class-main.php

```diff
@@ -149,7 +202,8 @@ public function on_account_protection_activation(): void {
 	 * @return void
 	 */
 	public function on_account_protection_deactivation(): void {
-		// Deactivation logic can be added here
+		$this->unregister_password_detection_hooks();
+		$this->unregister_strong_passwords_hooks();
```

Comment:
What is the impact of unregistering hooks on deactivation here? My initial thought is that with actions and filters being registered per request, once the feature/module is deactivated, the hooks shouldn't be registered on subsequent requests so we wouldn't need to do something like this 🤔

Reply:
I originally determined that unregistering hooks was the solution to fixing when the email was firing twice when we had a dedicated option per feature and were dynamically disabling registering certain actions. It was necessary at that point since the module remained active, but I think you're right now that its either on or off, its probably irrelevant. Like the activation method, I think we can now just remove the deactivation method also for now since we don't appear to need either.